src/app/services/tasksApi.ts

```diff
@@ -19,7 +32,33 @@ export const tasksApi = api.injectEndpoints({
             query: () => MINE_TASKS_URL,
             providesTags: ['Mine_Tasks'],
         }),
+        addTask: builder.mutation<TasksCreateMutationResponse, Partial<task>>({
+            query: (task: task) => ({
+                url: TASKS_URL,
+                method: 'POST',
+                body: task,
+            }),
+        }),
+        updateTask: builder.mutation<void, TaskRequestPayload>({
+            query: (task: TaskRequestPayload) => ({
+                url: `tasks/${task.id}`,
+                method: 'PATCH',
+                body: task.task,
+            }),
+            invalidatesTags: (_result, _err, { id }) => [
+                {
+                    type: 'Tasks',
+                    id,
+                },
+            ],
+        }),
     }),
+    overrideExisting: true,
```

Comment:
Nit: why you used overrideExisting here?? Is this common term in rtk query

Reply:
Please read: https://redux-toolkit.js.org/rtk-query/usage/code-splitting

Reply:
Override true is an optional parameter, which is by default false, could you specify the reason to override the task API?
![image](https://github.com/Real-Dev-Squad/website-status/assets/57758447/0824f307-abe2-48a4-bc83-32c10e08adc1)

Reply:
Due to hot reloading, I was getting a warning that already an end point exists (similar to the one mentioned in the screenshot you added). 
Hence, I searched about this and it was recommended to use `overrideExisting: true` to avoid issues when hot reload is applied.